Studies/source/VariableDistribution.cxx

```diff
@@ -0,0 +1,226 @@
+                double ellipse_cut = pow(event.SVfit_p4.mass()-116,2)/pow(35.,2) + pow(bb.mass()-111,2)/pow(45.,2);
+                if (ellipse_cut>1){
+                    current_entry++;
+                    continue;
+                }
+                vars.AddEvent(event, entry.mass, entry.weight);
+                current_entry++;
+                tot_entries++;
+            }
+            std::cout << entry << " number of events: " << tot_entries << std::endl;
+        }
+        auto stop = clock::now();
+        std::cout<<"secondi: "<<std::chrono::duration_cast<std::chrono::seconds>(stop - start).count()<<std::endl;
+        sample_vars = vars.GetSampleVariables();
+
+        outfile->mkdir("Distribution1D");
+        auto directory_distribution1d = outfile->GetDirectory("Distribution1D");
+        outfile->mkdir("Distribution2D");
+        auto directory_distribution2d = outfile->GetDirectory("Distribution2D");
+        for (const auto& sample: sample_vars){
+            start = clock::now();
+            std::string mass;
+            if (sample.first == Bkg) mass = "Bkg";
+            else if (sample.first == Signal_SM) mass = "SM";
+            else mass = std::to_string(sample.first);
+            std::cout<<"-----"<<mass<<"-----"<<std::endl;
+            std::cout<<"Distribution1D"<<std::endl;
+            directory_distribution1d->mkdir((mass).c_str());
+            auto directory_mass1d = directory_distribution1d->GetDirectory((mass).c_str());
+            Histo1D(sample.second, directory_mass1d);
+            stop = clock::now();
+            std::cout<<"secondi: "<<std::chrono::duration_cast<std::chrono::seconds>(stop - start).count()<<std::endl;
+            start = clock::now();
+            std::cout<<"Distribution2D"<<std::endl;
+            directory_distribution2d->mkdir((mass).c_str());
+            auto directory_mass2d = directory_distribution2d->GetDirectory((mass).c_str());
+            Histo2D(sample.second, directory_mass2d);
+            stop = clock::now();
+            std::cout<<"secondi: "<<std::chrono::duration_cast<std::chrono::seconds>(stop - start).count()<<std::endl;
+        }
+
+        auto stop_tot = clock::now();
+        std::cout<<"secondi totali: "<<std::chrono::duration_cast<std::chrono::seconds>(stop_tot - start_tot).count()<<std::endl;
+    }
+private:
+    Arguments args;
+    SampleEntryCollection samples;
+    std::shared_ptr<TFile> outfile;
+    bool split_training_testing;
+    std::mt19937 gen;
+    std::uniform_int_distribution<> test_vs_training;
+    MvaVariables::VarNameSet enabled_vars, disabled_vars;
+    MvaVariablesStudy vars;
+};
+}
+
+PROGRAM_MAIN(analysis::MvaClassification, Arguments) // definition of the main program function
+// /run.sh VariableDistribution --input_path ~/Desktop/tuples --output_file VariableDistribution_eTau.root --cfg_file hh-bbtautau/Studies/config/mva_config.cfg --tree_name eTau
```

Comment:
I think it will be more handy to put this command line into README.md in the same directory as the program.